rllib/algorithms/algorithm_config.py

```diff
@@ -6164,18 +6160,59 @@ def __init__(self, algo_class=None):
     def learners(
         self,
         *,
+        learner_class: Optional[Type["Learner"]] = NotProvided,
```

Comment:
I like this change. Makes more sense like this. We have `env_runner_class` under `env_runners`, `prelearner_class` under `offline_data`.

Reply:
Exactly! Algorithm and AlgorithmConfig need so much more cleanup, though :(